infrastructure/export.py

```diff
@@ -236,4 +255,43 @@ def refresh_views():
                     AS related
                 ON p.id = related.project_id;
             '''.format(status_cases=status_cases, **case_statements))
+        cursor.execute('''
+            CREATE OR REPLACE VIEW infrastructure_power_plant_export_view AS
+                SELECT pp.id, 
+                pp.name,
+                array_to_string(array_agg(icv.countries::text), ','::text, 'NULL'::text) as countries,
```

Comment:
I don't think these ``array_to_string`` and ``array_agg`` functions are needed here. They're performed in the underlying views. Just the column name should work.